src/components/OperationsBar/index.js

```diff
@@ -0,0 +1,79 @@
+import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
+import { Dropdown } from 'react-bootstrap';
+
+import Operation from './Operation';
+
+export default class OperationsBar extends PureComponent {
+  static propTypes = {
+    operations: PropTypes.arrayOf(PropTypes.shape({
+      style: PropTypes.oneOf(['primary', 'success', 'info', 'warning', 'danger', 'link']),
+      title: PropTypes.string.isRequired,
+      dropdown: PropTypes.bool.isRequired
```

Comment:
TBD why required? you treat empty value as false further in code.
It will not force setting explicit 'false' in VIEW/container.js for every standard button.